src/project-manager.ts

```diff
@@ -796,7 +798,11 @@ export class ProjectConfiguration {
 	 * the whole project or in some files
 	 */
 	syncProgram(): void {
-		this.program = this.getService().getProgram();
+		try {
+			this.program = this.getService().getProgram();
+		} catch (e) {
+			this.logger.error(`Cannot create program object for ${this.rootFilePath}`, e);
```

Comment:
I think so. We got a broken code, why should we die because of other sins?  :smile: